src/pages/home/report/AnimatedEmptyStateBackground.tsx

```diff
@@ -12,6 +12,10 @@ import CONST from '@src/CONST';
 const IMAGE_OFFSET_X = 30;
 const IMAGE_OFFSET_Y = 20;
 
+// This is necessary so we don't send the entire CONST object to the worklet which could lead to performance issues
+// https://docs.swmansion.com/react-native-reanimated/docs/guides/worklets/#capturing-closure
+const ANIMATION_GYROSCOPE_VALUE = CONST.ANIMATION_GYROSCOPE_VALUE;
```

Comment:
We no longer need this workaround since we already use `react-native-reanimated` to 3.16.4.

Reply:
It's not needed but we should do it. `CONST` is a huge object and it's creating unnecessary work.

Reply:
Okay, sounds good.